heartbeat/monitors/monitor.go

```diff
@@ -132,6 +172,20 @@ See https://www.elastic.co/guide/en/beats/heartbeat/current/configuration-heartb
 	return m, nil
 }
 
+func (m *Monitor) configHash() (uint64, error) {
+	unpacked := map[string]interface{}{}
+	err := m.config.Unpack(unpacked)
+	if err != nil {
+		return 0, err
+	}
+	hash, err := hashstructure.Hash(unpacked, nil)
```

Comment:
Make sure to compare this code with the hashing code in libbeat reloading. I remember there were some problems in the beginning.

Reply:
Hmmmm, we use this same methodology for autodiscover, so I think it's correct. I did verify it myself testing it against different configs and watching the values change.

Reply:
Perfect. Just wanted to double check.